example/src/screens/CustomHtmlUI.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react';
+import { View, Platform, StyleSheet } from 'react-native';
+import { useFocusEffect } from '@react-navigation/native';
+import {
+  Event,
+  usePlayer,
+  PlayerView,
+  SourceType,
+} from 'bitmovin-player-react-native';
+import { useTVGestures } from '../hooks';
+
+function prettyPrint(header: string, obj: any) {
+  console.log(header, JSON.stringify(obj, null, 2));
+}
+
+export default function BasicPlayback() {
+  useTVGestures();
+
+  const player = usePlayer({
+    styleConfig: {
+      playerUiCss:
+        'https://cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.css',
```

Comment:
@rolandkakonyi This is basically the same as the default interface, is there an example bitmovin hosted version somewhere that does something slightly different?  The android example use a `file://` URI and commits it to the repo, but that didn't sit right with me for some reason.

Reply:
@joornby-angel We don't have anything other than the default ones hosted right now. If you could share a CSS file with some modifications that you would feel useful to demonstrate this feature, we could put it on our CDN and use it for this example application.

Reply:
@joornby-angel Thanks for the patience. You can use this URL and commit it to the repository for the example screen: `https://storage.googleapis.com/bitmovin-player-cdn-origin/player/ui/ui-customized-sample.css`